docs/fuzzing.md

```diff
@@ -130,6 +152,22 @@ Dirs:
 * PULL_DIR  - out/the_fuzzer/queue
 * CRASH_DIR - out/the_fuzzer/crashes
 
+Synchronization:
```

Comment:
Is it describing what happens under the hood or what the user has to do?

In the first case, we might rename this e.g. `Actions done by DeepState for synchronization:` (?) or highlight that its internal info(?)

Reply:
same below

Reply:
Its mix of both: sometimes I write that AFL do smthing and sometimes that executor do smthing. But thats more like internal/advanced info. Will describe that a bit more maybe